src/magic_folder/snapshot.py

```diff
@@ -420,15 +426,17 @@ def create_snapshot_from_capability(snapshot_cap, tahoe_client):
         snapshot_json = yield tahoe_client.download_capability(snapshot_cap)
         snapshot = json.loads(snapshot_json)[1]["children"]
 
-        # create SnapshotAuthor
-        author_cap = snapshot["author"][1]["ro_uri"]
-        author_json = yield tahoe_client.download_capability(author_cap)
-        snapshot_author = json.loads(author_json)
-
-        author = create_author_from_json(snapshot_author)
+        # NB: once we communicate authors + public-keys some other
+        # way, we could iterate all our known authors and attempt to
+        # verify a signature against the snapshot + metdata
+        # capabilities at this point .. that might be better anyway?
+        # (A key advantage is not even trying to deserialize anything
+        # that's not verified by a signature).
+        metadata_cap = snapshot["metadata"][1]["ro_uri"]
+        author_signature = snapshot["metadata"][1]["metadata"]["magic_folder"]["author_signature"]
```

Comment:
Demeter is weeping.  It's a Tahoe-LAFS data structure though, so too bad for Demeter, I guess?

Reply:
Tahoe actually stores this stuff differently -- but yeah, we're consistent with the JSON output of the Tahoe WebUI here.